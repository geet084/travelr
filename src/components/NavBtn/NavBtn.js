import React from 'react'
import { NavLink } from 'react-router-dom'
import PropTypes from 'prop-types';

export const NavBtn = ({ name, path }) => {
  
  return (
    <li>
      <NavLink
        to={path.toLowerCase()}
        exact={true}
        activeClassName="active"
      >
        {name}
      </NavLink>
    </li>
  )
}

NavBtn.propTypes = {
  name: PropTypes.string,
  path: PropTypes.string,
}

NavBtn.defaultProps = {
  name: '',
  path: '',
}

export default NavBtn;