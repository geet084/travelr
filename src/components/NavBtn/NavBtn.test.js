import React from 'react';
import { shallow } from 'enzyme';
import NavBtn from './NavBtn';

describe('NavBtn', () => {
  let wrapper;
  const mockName = 'name';
  const mockPath = '/'

  beforeEach(() => {
    wrapper = shallow(
      <NavBtn name={mockName} path={mockPath} />
    )
  })
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

})