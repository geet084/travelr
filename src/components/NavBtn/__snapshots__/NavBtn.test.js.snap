// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBtn should match snapshot 1`] = `
<li>
  <Link
    replace={false}
    to="/"
  >
    name
  </Link>
</li>
`;
